2025-07-11 17:19:50,939 - database - INFO - База данных успешно инициализирована
2025-07-11 17:19:55,162 - __main__ - INFO - Google Sheet инициализирована
2025-07-11 17:19:58,898 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 17:19:58,898 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 17:19:58,899 - __main__ - INFO - Бот запущен...
2025-07-11 17:20:02,693 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 17:25:24,565 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 17:25:24,567 - TeleBot - ERROR - Break infinity polling
2025-07-11 17:25:44,310 - database - INFO - База данных успешно инициализирована
2025-07-11 17:25:48,295 - __main__ - INFO - Google Sheet инициализирована
2025-07-11 17:25:52,853 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 17:25:52,854 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 17:25:52,855 - __main__ - INFO - Бот запущен...
2025-07-11 17:25:58,137 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 17:34:36,659 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 17:34:36,659 - TeleBot - ERROR - Break infinity polling
2025-07-11 18:05:03,465 - database - INFO - База данных успешно инициализирована
2025-07-11 18:05:07,851 - __main__ - INFO - Google Sheet инициализирована
2025-07-11 18:05:12,404 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 18:05:12,404 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 18:05:12,405 - __main__ - INFO - Бот запущен...
2025-07-11 18:05:12,940 - TeleBot - ERROR - Infinity polling exception: argument of type 'int' is not iterable
2025-07-11 18:05:12,941 - TeleBot - ERROR - Exception traceback:
Traceback (most recent call last):
  File "C:\Users\nekit\AppData\Roaming\Python\Python311\site-packages\telebot\__init__.py", line 1110, in infinity_polling
    self.polling(non_stop=True, timeout=timeout, long_polling_timeout=long_polling_timeout,
  File "C:\Users\nekit\AppData\Roaming\Python\Python311\site-packages\telebot\__init__.py", line 1198, in polling
    self.__threaded_polling(non_stop=non_stop, interval=interval, timeout=timeout, long_polling_timeout=long_polling_timeout,
  File "C:\Users\nekit\AppData\Roaming\Python\Python311\site-packages\telebot\__init__.py", line 1273, in __threaded_polling
    raise e
  File "C:\Users\nekit\AppData\Roaming\Python\Python311\site-packages\telebot\__init__.py", line 1235, in __threaded_polling
    self.worker_pool.raise_exceptions()
  File "C:\Users\nekit\AppData\Roaming\Python\Python311\site-packages\telebot\util.py", line 150, in raise_exceptions
    raise self.exception_info
  File "C:\Users\nekit\AppData\Roaming\Python\Python311\site-packages\telebot\util.py", line 93, in run
    task(*args, **kwargs)
  File "C:\Users\nekit\AppData\Roaming\Python\Python311\site-packages\telebot\__init__.py", line 9759, in _run_middlewares_and_handler
    result = handler['function'](message)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "O:\CODE\TG BOTS\MANIK BOT — копия\bot.py", line 612, in admin_panel
    if message.chat.id not in ADMIN_CHAT_IDS:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'int' is not iterable

2025-07-11 18:05:17,161 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 18:08:08,725 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 18:08:08,725 - TeleBot - ERROR - Break infinity polling
2025-07-11 18:08:15,259 - database - INFO - База данных успешно инициализирована
2025-07-11 18:08:18,466 - __main__ - INFO - Google Sheet инициализирована
2025-07-11 18:08:23,021 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 18:08:23,022 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 18:08:23,023 - __main__ - INFO - Бот запущен...
2025-07-11 18:08:26,312 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 18:18:30,581 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 18:28:35,101 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 18:38:40,084 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 18:48:44,249 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 18:57:28,849 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 18:57:28,850 - TeleBot - ERROR - Break infinity polling
2025-07-11 18:57:37,147 - database - ERROR - Ошибка инициализации БД: no such column: reminder_sent
2025-07-11 19:02:45,971 - database - INFO - Добавлен столбец reminder_sent
2025-07-11 19:02:45,990 - database - INFO - База данных успешно инициализирована
2025-07-11 19:02:48,186 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:02:50,104 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:02:50,104 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 19:02:50,105 - __main__ - INFO - Бот запущен...
2025-07-11 19:02:52,058 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:04:48,989 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:05:58,866 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:06:17,771 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:06:35,991 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:06:40,453 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:06:45,423 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:09:44,115 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:09:50,580 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 19:09:50,581 - TeleBot - ERROR - Break infinity polling
2025-07-11 19:09:58,628 - database - INFO - База данных успешно инициализирована
2025-07-11 19:10:00,602 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:10:02,566 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:10:02,566 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 19:10:02,571 - __main__ - INFO - Бот запущен...
2025-07-11 19:10:02,902 - __main__ - INFO - Отправлено напоминание клиенту 582182474
2025-07-11 19:10:03,042 - __main__ - INFO - Отправлено напоминание клиенту 444796170
2025-07-11 19:10:04,489 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:10:30,407 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:13:48,146 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:13:50,326 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:13:55,204 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:14:51,491 - __main__ - ERROR - Ошибка доступа к Google Sheets: Zapis
2025-07-11 19:16:16,035 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 19:16:16,035 - TeleBot - ERROR - Break infinity polling
2025-07-11 19:16:21,569 - database - INFO - База данных успешно инициализирована
2025-07-11 19:16:24,221 - __main__ - INFO - Google Sheet инициализирована
2025-07-11 19:16:28,858 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 19:16:28,859 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 19:16:28,871 - __main__ - INFO - Бот запущен...
2025-07-11 19:16:29,215 - __main__ - INFO - Отправлено напоминание клиенту 582182474
2025-07-11 19:16:29,343 - __main__ - INFO - Отправлено напоминание клиенту 582182474
2025-07-11 19:16:32,857 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 19:17:17,688 - __main__ - ERROR - Ошибка показа календаря: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2025-07-11 19:26:36,371 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 19:27:08,787 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 19:27:08,787 - TeleBot - ERROR - Break infinity polling
2025-07-11 19:27:50,991 - database - INFO - База данных успешно инициализирована
2025-07-11 19:27:53,548 - __main__ - INFO - Google Sheet инициализирована
2025-07-11 19:27:57,445 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 19:27:57,446 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 19:27:57,447 - __main__ - INFO - Бот запущен...
2025-07-11 19:28:00,701 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 19:38:03,737 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 19:39:51,852 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 19:39:51,856 - TeleBot - ERROR - Break infinity polling
2025-07-11 19:42:37,854 - database - INFO - База данных успешно инициализирована
2025-07-11 19:42:40,792 - __main__ - INFO - Google Sheet инициализирована
2025-07-11 19:42:45,382 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 19:42:45,382 - __main__ - INFO - Google Sheets инициализирована
2025-07-11 19:42:45,383 - __main__ - INFO - Бот запущен...
2025-07-11 19:42:45,655 - __main__ - INFO - Отправлено напоминание клиенту 582182474
2025-07-11 19:42:48,790 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-11 19:43:31,660 - TeleBot - ERROR - Infinity polling: polling exited
2025-07-11 19:43:31,660 - TeleBot - ERROR - Break infinity polling
2025-07-12 15:46:34,388 - database - INFO - База данных успешно инициализирована
2025-07-12 15:46:36,517 - __main__ - INFO - Google Sheet инициализирована
2025-07-12 15:46:40,408 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-12 15:46:40,409 - __main__ - INFO - Google Sheets инициализирована
2025-07-12 15:46:40,409 - __main__ - INFO - Бот запущен...
2025-07-12 15:46:43,712 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
2025-07-12 15:56:47,637 - __main__ - INFO - Полная синхронизация с Google Sheets выполнена
